tests/test_fixed_price_oracle.py

```diff
@@ -0,0 +1,49 @@
+from decimal import (
+    Decimal,
+)
+
+from tests.constants import (
+    DEADLINE
+)
+
+DAI_PRICE = 1010000000000000000 #1.01*10**18
+
```

Comment:
Can we add a test to make sure that `normalized_token_prices` and `poolSize` will fail if the DAI price is < 0.95 or > 1.05 ?

Reply:
sure